### PR TITLE
Added filtering pokemons which you wish to show

### PR DESCRIPTION
**Pull request by fess89**
The settings screen now has an item "Pokemons to show" where the user can select pokemons which need to be shown (all are on by default).

### CODE REVIEW

**Comment by fess89** [comment]
filtering

File: app/src/main/java/com/omkarmoghe/pokemap/views/map/MapWrapperFragment.java [hunk 1]
```diff
@@ -309,47 +320,52 @@ private void setPokestopsMarkers(final Collection<Pokestop> pokestops){
...
+                    PokemonIdOuterClass.PokemonId pokemonId = poke.getPokemonId();
+                    if (showablePokemonIDs.contains(pokemonId)) {
+
+                        //Showing images using glide
+                        Glide.with(getActivity())
+                                .load("http://serebii.net/pokemongo/pokemon/" + getCorrectPokemonImageId(pokemonId.getNumber()) + ".png")
+                                .asBitmap()
+                                .skipMemoryCache(false)
+                                .diskCacheStrategy(DiskCacheStrategy.ALL)
+                                .into(new SimpleTarget<Bitmap>(120, 120) { // Width and height FIXME: Maybe get different sizes based on devices DPI? this need tests
```
Comment (s7092910) [inline app/src/main/java/com/omkarmoghe/pokemap/views/map/MapWrapperFragment.java]: Missing Change from PR #169

**Comment by fess89** [comment]
@s7092910 thanks for pointing that out, added the change

**Comment by jLynx** [comment]
Would be cool if you had a little image of each pokemon next to the name

**Comment by fess89** [comment]
@DarkN3ss61 I could try, for now it is just the standard preferences list dialog which does not have images

**Comment by Anu6is** [comment]
The plural of Pokemon should be Pokemon.

**Comment by fess89** [comment]
@Anu6is you are correct, fixed

**Comment by comann** [comment]
Awesome feature. Please update to dev tip.

**Comment by omkarmoghe** [comment]
Dope!

**Comment by fess89** [comment]
resolved merge conflicts, added pokemon icons to the filter list
filtering-with-icons

**Comment by comann** [comment]
Does this still use AndroidAnnotations?

**Pull request merged**